.babelrc

```diff
@@ -13,6 +13,7 @@
     ]
   ],
   "plugins": [
+    "babel-plugin-lodash",
```

Comment:
This was removed on purpose in https://github.com/opencollective/opencollective-frontend/pull/9161 because the plugin is deprecated. It slightly improves the bundle size, but at the risk of running deprecated code.

Reply:
A mitigating factor is that this is a plugin that is executed on the build phase only. We might remove the need for it eventually if we move to lodash ESM which supports treeshaking. Its a nice reduction though as lodash was one of the biggest packages imported into the main bundle.

Reply:
In the abovementioned PR, we measured that the gain was around 15kb. In my opinion, the risk of running a non-updated version of lodash (which may include bugs, including security) is not worth the save. Running different versions of lodash between development and production (build) also seems dangerous.

Reply:
I got it wrong, the plugin works on the transpilation phase, so it affects both dev and prod builds, its just that package is not a runtime dependency. The plugin changes `import { get } from 'lodash'` to `import get from 'lodash/get'`, so it does change the running version of lodash that way, just the import style.

I tested another change using `lodash-es` which supports tree-shaking and does not use this deprecated plugin and I push into another PR for review. Reverting this here.

Digging into this I was surprised to see that lodash does not get a release in over 2 years now. :(